applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -296,8 +296,9 @@ See :ref:`Building with overlays <building_with_overlays>` for information on ho
 
 .. note::
 
-   This application supports :ref:`ug_bootloader`, which is disabled by default.
-   To enable the immutable bootloader, set ``CONFIG_SECURE_BOOT=y``.
+   This application supports the :ref:`ug_bootloader` (also called immutable bootloader), which has been enabled by default since the v2.0.0 release.
```

Comment:
```suggestion
   This application supports the :ref:`ug_bootloader` (also called immutable bootloader), which has been enabled by default since the |NCS| v2.0.0 release.
```